backend/src/test/java/de/kathahuth/gymast/controller/FamilyControllerTest.java

```diff
@@ -40,7 +40,7 @@ private Family createPushUpFamily() {
         return Family.builder()
                 .id("push-up-family")
                 .name("Push Up Family")
-                .childrenIds(Arrays.asList("push-up-category"))
+                .childrenNames(Arrays.asList("Push Up Category"))
```

Comment:
Maybe this is only the case in the tests: don't you want to opt for a uniform nomentclature? - ' ', CamelCase, pascalCase...